projects/igniteui-angular/src/lib/action-strip/action-strip.component.ts

```diff
@@ -4,16 +4,25 @@ import {
     HostBinding,
     Input,
     Renderer2,
-    TemplateRef,
     ViewContainerRef,
     Optional,
     Inject,
     ContentChildren,
     QueryList,
-    ViewChild
+    ViewChild,
+    ElementRef,
+    TemplateRef
 } from '@angular/core';
 import { DisplayDensityBase, DisplayDensityToken, IDisplayDensityOptions } from '../core/density';
 import { IgxDropDownComponent } from '../drop-down';
+import { IgxGridBaseDirective, IgxRowDirective } from '../grids';
+
+export interface IActionStripContext {
+    element: ElementRef<any>;
+    grid?: IgxGridBaseDirective;
```

Comment:
Erm, shouldn't this be generic for the strip as a whole? I.e. Not related to the grid in any way?

Reply:
Decided to scrap the `IActionStripContext` and use `any` for the context, as the context can be anything.